tests/screenshot/apps/components/Popup.js

```diff
@@ -4,6 +4,14 @@ import React from 'react';
 const PopupTests = [
 	<Popup closeButton open>Popup!</Popup>,
 	<Popup centered open>Popup!</Popup>,
+	<Popup open duration="short">Popup!</Popup>,
+	<Popup open noAnimation>Popup!</Popup>,
+	<Popup open type="fade">Popup!</Popup>,
+	<Popup open position="bottom">Popup!</Popup>,
+	<Popup open position="center">Popup!</Popup>,
```

Comment:
It will be the same as <Popup open duration="short">Popup!</Popup>,.

Reply:
True, I will remove it.